cedar-policy/src/api.rs

```diff
@@ -2691,18 +2691,25 @@ impl TemplateResourceConstraint {
 
 /// Unique ids assigned to policies and templates.
 ///
-/// A `PolicyId` can can be constructed using [`PolicyId::from_str`] or by
+/// A [`PolicyId`] can can be constructed using [`PolicyId::from_str`] or by
```

Comment:
I left this unlinked earlier because we're already in the doc comment for `PolicyId`.